fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/models/FedoraResource.java

```diff
@@ -156,18 +149,18 @@ RdfStream getTriples(final IdentifierConverter<Resource, FedoraResource> idTrans
                          final Set<? extends TripleCategory> contexts);
 
     /**
-     * Get the JCR Base version for the node
+     * Get the base version for the node
      *
      * @return base version
      */
-    public Version getBaseVersion();
+    FedoraResource getBaseVersion();
 
     /**
-     * Get JCR VersionHistory for the node.
+     * Get a stream of version labels associated with this resource
```

Comment:
Actually, we need to review _all_ of the names for these methods. If you'd like, though, I could change the method name to `getVersionIdentifiers`.